src/constants.ts

```diff
@@ -9,4 +9,7 @@ export const HEAD_FINALITY_CHECKPOINTS_PATH = ETH_API_PREFIX + "/beacon/states/h
 export const STATE_PATH = ETH_API_PREFIX + "/debug/beacon/states/";
 export const WS_EPOCH_PATH = ETH_API_PREFIX + "/lodestar/ws_epoch/";
 
-export const CID_FILE_PATH = "./.cid";
\ No newline at end of file
+export const CID_FILE_PATH = "./.cid";
+export const WS_STATE_DATA_FILE_PATH = "./weakSubjectivityState.json";
+
+export const WAITING_MSG = "Waiting for finalized checkpoints...";
```

Comment:
Why is this a constant instead of just logging it directly?

Reply:
because i'm logging the message more than once

Reply:
I would just copy the message multiple times, It's counterproductive to have to go the the constants file to see what are you logging